app/views/backstage/labs/edit.html.haml

```diff
@@ -8,38 +8,33 @@
     .edit-lab
 
     %fieldset
-      %legend Basic Details
-      .row
-        .col-lg-12
-          %h5
-            Name, description and slug of your Lab
-          = f.input :name
-          = f.input :blurb
-          = f.input :description
-          = f.input :slug
-      .row
-        .col-lg-12
-          %h5
-            Kind of Lab
-          %p
-            What kind of Fab Lab is your Lab?
-          - Lab::kinds.each do |value, index|
-            .col-lg-2
-              .radio-inline
-                %span
-                  = f.radio_button :kind, value
-                  %label.radio{style: "width: 160px;"}
-                    = value.humanize.titleize
+      .text-large Basic Details
+      .text-medium
+        Name, description and slug of your Lab
+      = f.input :name
+      = f.input :blurb
+      = f.input :description
+      = f.input :slug
+      .text-medium
+        Kind of Lab
+      .hint
+        What kind of Fab Lab is your Lab?
+        .row
+        - Lab::kinds.each do |value, index|
+          .col-lg-2
+            %span.form-check.mb-0.mt-3
+              %div.form-check-input.mt-0
+                = f.radio_button :kind, value
+              %label.radio.form-check-label{style: "width: 160px;"}
+                = value.humanize.titleize
 
     %fieldset
-      %legend
+      .text-large
```

Comment:
Why is text-large better then legend?

Reply:
Sometimes he uses legend and sometimes he uses text-large. I simply choose text-large to keep it consistent. For now, because h1 & h2 is taken, I need to double-check if I can remove text-large :D